src/fshtypes/FshCanonical.ts

```diff
@@ -0,0 +1,12 @@
+import { FshEntity } from './FshEntity';
+
+export class FshCanonical extends FshEntity {
+  public sdType: string;
```

Comment:
What is `sdType` for?

Reply:
I removed it and the tests all still passed, so... maybe it was just a copy/paste thing?

Reply:
Yeah I think it was a copy/paste issue. Thanks for catching it.